server.js

```diff
@@ -18,6 +20,30 @@ const app = next({ dev });
 
 const handle = app.getRequestHandler();
 
+const checkStatus = response => response.ok ? response.json() : Promise.reject(response.json());
+
+function sendContactToHubSpot({ firstname, lastname, email, phone }) {
+  const postData = querystring.stringify({
+    firstname: firstname.value,
+    lastname: lastname.value,
+    email: email.value,
+    phone: phone.value.toString(),
+  });
+  const options = {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/x-www-form-urlencoded; charset=utf-8',
+      'Content-Length': postData.length,
+    },
+  };
+  const hubUrl = `https://forms.hubspot.com/uploads/form/v2/2769865/8552b6b3-5f76-4f18-a390-48b8f418fedc?${postData}`;
+
+  fetch(hubUrl, options)
+    .then(checkStatus)
+    .then(response => console.log(response))
```

Comment:
@dmytropanontko , what will be output here? Is it usefule?

Reply:
@epaminond I forgot remove that